sparse/impl/KokkosSparse_sptrsv_symbolic_impl.hpp

```diff
@@ -285,8 +285,7 @@ void lower_tri_symbolic(ExecSpaceIn& space, TriSolveHandle& thandle,
       check_count = 0;  // reset
       Kokkos::parallel_reduce(
           "check_count device",
-          Kokkos::RangePolicy<Kokkos::DefaultExecutionSpace>(
-              0, dnodes_per_level.extent(0)),
+          Kokkos::RangePolicy<ExecutionSpace>(0, dnodes_per_level.extent(0)),
```

Comment:
I would have expected that the `space` instance would be passed to the `RangePolicy` here?